recipes/arcas-hla/meta.yaml

```diff
@@ -1,31 +1,36 @@
-{% set version = "0.5.0" %}
+{% set version = "0.6.0" %}
 
 package:
   name: arcas-hla
   version: {{ version }}
 
 source:
   url: https://github.com/RabadanLab/arcasHLA/archive/v{{ version }}.tar.gz
-  sha256: fb17b3ba74facf4edb3329099b158910bdc7c4e73074efab9880d4136a0d4e93
+  sha256: 75bef1a5596a9dfabccb901420f5d852d6d8a15d9c676e4c94787b8d25dd708b
 
 build:
-  number: 3
+  number: 0
   noarch: generic
+  run_exports:
+    - {{ pin_subpackage("arcas-hla", max_pin="x.x") }}
 
 requirements:
   run:
-    - coreutils
-    - bedtools >=2.27.1
-    - biopython <=1.77
-    - git
-    - git-lfs
-    - kallisto ==0.44.0
-    - numpy
-    - pandas
-    - pigz
-    - python >=3.6.1
-    - samtools >=1.9
-    - scipy ==1.1.0
+    - bioconda::bedtools ==2.31.1
```

Comment:
please do not specify the channels here and avoid the strict pinning they are usually not needed, same for samtools and kallisto

Reply:
Thanks! Removed channels. I'd like to keep `kallisto` pinned because of a method that got deprecated. Other deps should be fine to unpin.